crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -1405,8 +1407,8 @@ impl EntityCommands<'_> {
     /// # bevy_ecs::system::assert_is_system(remove_character_system);
     /// ```
     #[track_caller]
-    pub fn despawn(self) -> Self {
```

Comment:
#14897 mistakenly made `despawn` return self

Reply:
For context, this wasn't a mistake, it was something acknowledged in the PR: https://github.com/bevyengine/bevy/pull/14897#issuecomment-2307615139